contracts/tests/ocr2.spec.ts

```diff
@@ -346,7 +346,7 @@ describe("ocr2", async () => {
         preInstructions: [
           await workspace.Store.account.transmissions.createInstruction(
             feed,
-            8 + 192 + 6 * 24
+            8 + 192 + 6 * 48
```

Comment:
We should properly document and type all these magic numbers.

Reply:
Extracted GH Issue:

- https://github.com/smartcontractkit/chainlink-solana/issues/219